pkg/configserver/secretstore.go

```diff
@@ -26,18 +26,25 @@ import (
 
 // configserverSecretStore is a SecretStore backed by the config server.
 type configserverSecretStore struct {
-	nodeConfig *nodeup.NodeConfig
+	nodeSecrets map[string]string
 }
 
 func NewSecretStore(nodeConfig *nodeup.NodeConfig) fi.SecretStore {
```

Comment:
Might be clearer to just take the `NodeSecrets` field, but that's just my 2c

Reply:
I agree. Thanks for the suggestion.